server/routes/users.js

```diff
@@ -10,4 +10,43 @@ router.get('/', function(req, res, next) {
   res.status(200).json({ user });
 });
 
+//Delete an user
+router.delete('/user/:id', async(req, res)=>{
+  try {
+    const userId = req.params.id;
+    const deletedUser = await User.findByIdAndDelete(userId);
+    if(!deletedUser){
+      return res.status(404).send({message: 'User not found'});//Error handling
+    }
+    res.json({ message: 'User deleted successfully', deletedUser });
+  } catch (error) {
+    console.log(error)
+    res.status(500).send({message: 'Internal server error'});//Error handling
+  }
+});
+
+//Update an user with new username, email, or password
+router.put('updateUser/:id', async(req, res)=>{
```

Comment:
for groups of related routes, it's generally much cleaner to stick with a common route name (ie all user related routes would be `/user`). I think it's easier to explain this in person, remind me at the meeting tomorrow to go over this

Reply:
Still needs to be updated